integration/resource_lacework_integration_gar_test.go

```diff
@@ -18,9 +18,10 @@ func TestIntegrationGARCreate(t *testing.T) {
 		terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
 			TerraformDir: "../examples/resource_lacework_integration_gar",
 			Vars: map[string]interface{}{
-				"client_id":      gcreds.ClientID,
```

Comment:
nit: I think underscores for field names is very pythonic and not idiomatic go. I would use camelcase and not snakecase.

Reply:
We're overriding the terraform variables here. https://www.terraform-best-practices.com/naming